tools/statvar_importer/mcf_file_util.py

```diff
@@ -353,8 +409,10 @@ def load_mcf_nodes(
         ...
       }
   """
+    if nodes is None:
+      nodes = {}
```

Comment:
Usually we should avoid reassigning method arguments, as references are passed by value and reassigning wont make it visible outside the method. Can we create a local reference (like local_nodes = nodes?nodes:{}) and work with it within the method?

Reply:
python seems to always pass by reference and changes are visible outside the function.
The caller is expected to get a dict object, even if empty.